packages/lab/stories/progress.qa.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import { ComponentMeta, Story } from "@storybook/react";
+
+import { CircularProgress, LinearProgress } from "@jpmorganchase/uitk-lab";
+import { QAContainer } from "docs/components";
+
+export default {
+  title: "Lab/Progress/QA",
+  component: CircularProgress,
+} as ComponentMeta<typeof CircularProgress>;
+
+export const AllExamplesGrid: Story = (props: {
+  className?: string;
+  imgSrc?: string;
+}) => {
```

Comment:
Maybe we could add `const { className, imgSrc } = props;` just to avoid repetition